mm2src/coins/qrc20/qrc20_tests.rs

```diff
@@ -55,7 +56,7 @@ fn check_tx_fee(coin: &Qrc20Coin, expected_tx_fee: ActualTxFee) {
 
 #[test]
 fn test_withdraw_impl_fee_details() {
-    Qrc20Coin::list_mature_unspent_ordered.mock_safe(|coin, _| {
+    Qrc20Coin::get_unspent_ordered_list.mock_safe(|coin, _| {
```

Comment:
Sorry, I didn't understand your question. If I'm not mistaken, neither `test_withdraw_impl_fee_details` nor `Qrc20Coin::get_unspent_ordered_list` haven't been changed.

What do you mean? 😅 
>  the functionality has been changed a little bit in this test function

Reply:
What I mean is ` list_mature_unspent_ordered` renamed into ` get_mature_unspent_ordered_list`, but you used `get_unspent_ordered_list ` . I wanted to know if you did it on purpose, I thought you might be confused on the function name. I guess you did it on purpose so no problem 👍

Reply:
I renamed it because there is similar method that returns `UnspentMap` of `address -> UnspentList`. So `map_mature_unspent_ordered` doesn't fit its purpose, so I had to rename all of those methods to `get_[all/mature]_unspent_ordered_[list/map]` 😁